l2-contracts/src/interfaces/IUnifiRewardsDistributor.sol

```diff
@@ -48,7 +51,56 @@ interface IUnifiRewardsDistributor {
     /**
      * @notice Registers the `claimer`'s address for the validator's BLS public keys
      * @param claimer The address of the claimer to register.
-     * @param params is an array of structs containing the G1 & G2 public keys of the validator, and a signature proving their ownership
+     * @param params is an array of structs containing the G1 & G2 public keys of the validator, and a signature
+     * proving their ownership
      */
     function registerClaimer(address claimer, PubkeyRegistrationParams[] calldata params) external;
+
+    /**
+     * @notice Claim the unclaimed rewards for multiple validators
+     * They all must have the same claimer set.
+     * @param token The token address to claim (use NATIVE_TOKEN for ETH)
+     * @param blsPubkeyHashes The hashes of the BLS public keys
+     * @param amounts The total cumulative earned amounts
+     * @param proofs The proofs of the claims
+     */
+    function claimRewards(
+        address token,
+        bytes32[] calldata blsPubkeyHashes,
+        uint256[] calldata amounts,
+        bytes32[][] calldata proofs
+    ) external;
+
+    /**
+     * @notice Set the Merkle root of the latest cumulative distribution
+     * @param newMerkleRoot The new Merkle root
+     */
+    function setNewMerkleRoot(bytes32 newMerkleRoot) external;
```

Comment:
I'd remove functions like this from the interface.

In no scenario, a normal user / or anybody who would copy the interface, will have access to this. In my mind, there is no need for them to know about this.